--- ./extras/systemd/glusterd.service.in.bis	2021-06-15 18:49:02.691517285 +0200
+++ ./extras/systemd/glusterd.service.in	2021-06-15 18:50:06.556183406 +0200
@@ -2,7 +2,7 @@
 Description=GlusterFS, a clustered file-system server
 Documentation=man:glusterd(8)
 Requires=@RPCBIND_SERVICE@
-After=network.target @RPCBIND_SERVICE@
+After=network.target iptables.target ip6tables.target @RPCBIND_SERVICE@
 Before=network-online.target

 [Service]
